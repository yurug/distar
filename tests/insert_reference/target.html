<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8" />
        <title>Online documentation</title>
    </head>
    <body>
      <h1> OCaml Documentation </h1>
      <p>
        let rec fac n = if n <= 0 then 1 else fac n * (n-1)
      </p>
    </body>
</html><!Doctype html>
