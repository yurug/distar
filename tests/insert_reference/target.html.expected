<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8" />
        <title>Online documentation</title>
    </head>
    <body>
      <h1> OCaml Documentation </h1>
      <p>
        <!-- distar:source.ml:3:1 -->
        let rec fac n = if n <= 0 then 1 else fac n * (n-1)
      </p>
    </body>
</html><!Doctype html>
